Oglle/dist/presence.js

```diff
@@ -10,7 +10,10 @@ presence.on("UpdateData", () => {
     };
     presenceData.startTimestamp = browsingStamp;
 
-    if(window.location.pathname.includes("")) {
+    if(window.location.pathname.includes()) {

```

Comment:
Why are you doing this?

Reply:
The start page is not showing up in the presence, I added this line and it started to work as expected!